tocrify/data/mets2hocr.yml

```diff
@@ -0,0 +1,37 @@
+chapter:
```

Comment:
Hi wrznr, 
oh, with the yaml file the problem was solved very elegantly. This file is easy to view and expand later. See https://dfg-viewer.de/strukturdatenset/ for more label attribute values.